### PR TITLE
finished create/destroy

### PR DESCRIPTION
**Pull request by pshim17**
1. Finished the create action
- tested using the spec files
- tested using Postman

2. Finished the destroy action
- tested using the spec files
- tested using Postman

3. Modified the Index action to match the formatting

### CODE REVIEW

File: spec/requests/api/v1/posters_request_spec.rb [hunk 1]
```diff
@@ -31,4 +33,48 @@
 
     expect(response).to be_successful
   end
-end
\ No newline at end of file
+
+  it "can create a new poster" do
```
Comment (jbleggi) [inline spec/requests/api/v1/posters_request_spec.rb:37]: Nice work remembering to update the rspec!!

File: app/controllers/api/v1/posters_controller.rb [hunk 2]
```diff
@@ -1,5 +1,60 @@
 class Api::V1::PostersController < ApplicationController
   def index
-    render json: Poster.all
+    posters = Poster.all
+
+    render json: posters.map { |poster|
```
Comment (jbleggi) [inline app/controllers/api/v1/posters_controller.rb:5]: This is well done! Let's move this to a serializer later in the project :)

**Review by jbleggi — APPROVED** [review]
Nice work Patrick! Off to a great start 👍

File: app/controllers/api/v1/posters_controller.rb (hunk 2 above)
Comment (pshim17) [reply app/controllers/api/v1/posters_controller.rb:5]: That's a great idea! I will need your help with the serializers!

**Pull request merged**